stubs/stripe/stripe/api_resources/abstract/createable_api_resource.pyi

```diff
@@ -6,10 +7,10 @@ from stripe.api_resources.abstract.api_resource import APIResource as APIResourc
 class CreateableAPIResource(APIResource):
     @classmethod
     def create(
-        cls,
+        cls: type[Self],
         api_key: Any | None = ...,
         idempotency_key: Any | None = ...,
         stripe_version: Any | None = ...,
         stripe_account: Any | None = ...,
         **params,
-    ): ...
+    ) -> Self: ...
```

Comment:
This I'm having trouble figuring out from looking at the source code, can you copy a REPL demonstration for me to see?

Reply:
```python
>>> type(stripe.Customer.create())
<class 'stripe.api_resources.customer.Customer'>
```

essentially all the resources inherit from `CreateableAPIResource` and that `.create` method is used to create the resource instance